hudi-common/src/main/java/org/apache/hudi/metadata/MetadataPartitionType.java

```diff
@@ -332,13 +332,13 @@ public String getIndexNameWithoutPrefix(HoodieIndexDefinition indexDefinition) {
    */
   public static boolean isGenericIndex(String metadataPartitionPath) {
```

Comment:
Let's rename this method and below one of `isGeneric*` to `isExpressionOrSecondary*`

Reply:
Addressed